modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/UserCardTag.java

```diff
@@ -0,0 +1,574 @@
+	public void setUserCard(UserCard userCard) {
+		_userCard = userCard;
+	}
+
+	public void setUserColorClass(String userColorClass) {
+		_userColorClass = userColorClass;
+	}
+
+	@Override
+	protected void cleanUp() {
+		super.cleanUp();
+
+		_actionDropdownItems = null;
+		_disabled = null;
+		_groupName = null;
+		_href = null;
+		_icon = null;
+		_imageAlt = null;
+		_imageSrc = null;
+		_inputName = null;
+		_inputValue = null;
+		_labels = null;
+		_name = null;
+		_selectable = null;
+		_selected = null;
+		_spritemap = null;
+		_subtitle = null;
+		_userCard = null;
+		_userColorClass = null;
+	}
+
+	@Override
+	protected String getHydratedModuleName() {
+		return "frontend-taglib-clay/UserCard";
+	}
+
+	@Override
+	protected Map<String, Object> prepareProps(Map<String, Object> props) {
+		props.put("actions", getActionDropdownItems());
+		props.put("description", getSubtitle());
+		props.put("disabled", isDisabled());
+		props.put("href", getHref());
+		props.put("id", getId());
+		props.put("inputName", getInputName());
+		props.put("inputValue", getInputValue());
+		props.put("userImageAlt", getImageAlt());
+		props.put("userImageSrc", getImageSrc());
+		props.put("userSymbol", getIcon());
+		props.put("labels", getLabels());
+		props.put("name", getName());
+		props.put("selectable", isSelectable());
+		props.put("selected", isSelected());
+		props.put("userDisplayType", getUserColorClass());
+
+		return super.prepareProps(props);
+	}
+
+	@Override
+	protected String processCssClasses(Set<String> cssClasses) {
+		Boolean selectable = isSelectable();
+
+		if ((selectable == null) || !selectable) {
+			cssClasses.add("card");
+		}
+
+		cssClasses.add("card-type-asset");
+		cssClasses.add("form-check");
+		cssClasses.add("form-check-card");
+		cssClasses.add("form-check-top-left");
+		cssClasses.add("user-card");
+
+		return super.processCssClasses(cssClasses);
+	}
+
+	@Override
+	protected int processStartTag() throws Exception {
+		super.processStartTag();
+
+		JspWriter jspWriter = pageContext.getOut();
+
+		jspWriter.write("<div class=\"card\"><div class=\"aspect-ratio ");
+		jspWriter.write("card-item-first\">");
+
+		Boolean disabled = isDisabled();
+		Boolean selectable = isSelectable();
+
+		if ((selectable != null) && selectable) {
+			jspWriter.write("<div class=\"custom-control custom-checkbox\">");
+			jspWriter.write("<label><input ");
+
+			if ((disabled != null) && disabled) {
+				jspWriter.write("disabled=\"disabled\" ");
+			}
+
+			String inputName = getInputName();
+
+			if (Validator.isNotNull(inputName)) {
+				jspWriter.write("name=\"" + inputName + "\" ");
+			}
+
+			String inputValue = getInputValue();
+
+			if (Validator.isNotNull(inputValue)) {
+				jspWriter.write("name=\"" + inputValue + "\" ");
+			}
+
+			Boolean selected = isSelected();
+
+			if ((selected != null) && selected) {
+				jspWriter.write("checked=\"checked\" ");
+			}
+
+			jspWriter.write("class=\"custom-control-input\" type=\"checkbox\"");
```

Comment:
Auto review: Sort this: First `class`, then `disabled`, `inputName`, `inputValue` and `selected` conditions, then `type` and `value`.